src/reports/models/aggregateReportManager.js

```diff
@@ -15,6 +16,7 @@ module.exports = {
 
 async function aggregateReport(report) {
     let stats = await databaseConnector.getStats(report.test_id, report.report_id);
+    const experiments = await getChaosExperiments(report.job_id);
```

Comment:
what happens if we are not in k8s env?

Reply:
it will not find experiments because when we insert we check the job platform